src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -261,14 +261,20 @@ export class SbbFormField implements ComponentInterface {
     }
   }
 
+  private _slotHasContent(name: string): boolean {
+    return !!this._element.querySelector(`[slot="${name}"]`);
+  }
+
   public render(): JSX.Element {
     return (
       <div class="sbb-form-field__space-wrapper">
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
         <div onClick={(event) => this._handleWrapperClick(event)} class="sbb-form-field__wrapper">
-          <div class="sbb-form-field__slot-container">
-            <slot name="prefix"></slot>
-          </div>
+          {this._slotHasContent('prefix') && (
```

Comment:
We have for that the the named slot handler which also takes care of changing events (see examples for full code snippets).
`  @State() private _namedSlots = createNamedSlotState('prefix');`
`  private _handlerRepository = new HandlerRepository(
    this._element,
    namedSlotChangeHandlerAspect((m) => (this._namedSlots = m(this._namedSlots)))
  ); `